src/sentry/search/eap/resolver.py

```diff
@@ -635,11 +636,12 @@ def resolve_aggregate_term(
             raise InvalidSearchQuery(f"Unknown operator: {term.operator}")
 
         kwargs = {"op": operator, "val": value}
-        aggregation_key = (
-            "conditional_aggregation"
-            if isinstance(proto_definition, AttributeConditionalAggregation)
-            else "aggregation"
-        )
+        aggregation_key = "aggregation"
+        if isinstance(proto_definition, AttributeConditionalAggregation):
+            aggregation_key = "conditional_aggregation"
+        elif isinstance(proto_definition, Column.BinaryFormula):
+            aggregation_key = "formula"
```

Comment:
nit
```suggestion
        if isinstance(proto_definition, AttributeConditionalAggregation):
            aggregation_key = "conditional_aggregation"
        elif isinstance(proto_definition, Column.BinaryFormula):
            aggregation_key = "formula"
        elif isinstance(proto_definition, Column.BinaryFormula):
            aggregation_key = "aggregation"
        else:
        	throw someerror
```

Reply:
Hmm, a few lines above we do
```
 if not isinstance(
            proto_definition,
            (AttributeAggregation, AttributeConditionalAggregation, Column.BinaryFormula),
        ):
            raise ValueError(f"{term.key.name} is not valid search term")
```
So should be fine as is I think?

Reply:
👍 made the change